ckanext/push_errors/tests/test_push_errors.py

```diff
@@ -0,0 +1,82 @@
+from unittest import mock
+import pytest
+from ckan.lib.helpers import url_for
+from ckan.tests import factories
+from ckanext.push_errors.blueprints import push_errors
+
+
+class TestPushErrorView:
+    """Tests for the push_errors blueprint"""
+
+    @mock.patch('ckanext.push_errors.blueprints.push_errors.log')
+    def test_unauthorized_no_user(self, mock_log, app):
+        """
+        Test that a 403 response is returned when no user is logged in
+        """
+        with app.flask_app.app_context():
+            with mock.patch.object(push_errors.toolkit, '_', lambda x: x):
+                client = app.test_client()
+                response = client.get('/push-error/test')
+
+        assert response.status_code == 403
+        assert b'Unauthorized to access this page' in response.data
+        mock_log.critical.assert_not_called()
+
+    @mock.patch('ckanext.push_errors.blueprints.push_errors.log')
+    def test_unauthorized_not_sysadmin(self, mock_log, app):
+        """
+        Test that a 403 response is returned when the user is not a sysadmin
+        """
+        user_with_token = factories.UserWithToken()
+        url = url_for('/push-error/test', msg='test')
+        auth = {"Authorization": user_with_token['token']}
+        response = app.get(url, headers=auth)
+
+        assert response.status_code == 403
+        assert b'Unauthorized to access this page' in response.data
+        mock_log.critical.assert_not_called()
+
+    @pytest.mark.ckan_config('ckanext.push_errors.url', 'http://example.com')
```

Comment:
```suggestion
```
If we have an URL in test.ini this is not required
This will only be required if we need a custom url to test something related with the URL
Remove this and all other if they are not required.